### ISSUE TITLE
Do you plain to develop this project?

### ISSUE THREAD

**Issue by zerkalica** [gcanti/tcomb-json-schema#24]
tcomb itself is js-only. But json-schema can be used in different environments (java/svagger, etc).

On current moment only [ajv](https://github.com/epoberezkin/ajv) supports many features of json-schema draft v4 and 5 (meta-schemas, oneOf, allOf, refs) and provides complex information about errors: format, schemaPath, keyword, etc.

But ajv is too fat for frontend: about 200 kb. Not a separation concern: doT templates, code generation, unnecessary for frontend speed optimizations, async support are monolythic parts of ajv.

I also try is-my-json-valid, jsen, schemasaurus. But all them are garbage, suitable only for true/false validation.

Tcomb - is good engine for scmema validators, provides best meta-information about errors. But tcomb-json-schema does not support some features of draft v4: oneOf/allOf, refs, metaschemas.

Do you plain to add this features?

**Comment by gcanti** [comment]
> Do you plan to add this features?

No, I don't. I don't use this library at work so currently is in maintenance mode. If anyone is willing to add those features, or become the new maintainer or fork this repo I will be glad to help out
